modules/admin_manual/pages/configuration/integration/ms-teams.adoc

```diff
@@ -0,0 +1,123 @@
+
+. Enable xref:configuration/server/index_php_less_urls.adoc[index.php less URL´s] on your web server. 
+
+== Microsoft
+
+=== Microsoft Azure and OpenID Connect
+
+// C&P from Github issue 3387 plus language fixes. Is Azure really the first thing to take care of? The app hasn't been generated yet.
+
+Follow the procedure described in
+xref:configuration/user/oidc/ms-azure-setup.adoc.adoc[Example Setup Using Microsoft Azure] to configure Microsoft Azure AD and OpenID Connect.
+
+=== Create Your Microsoft Teams App
+
+The following procedure creates a Microsoft Teams App configured for your environment and ready to use for your users.
+
+. In {appsource-url}[Microsoft Teams AppSource], search for *ownCloud Generator for Admins* and open the application.
+
+. Now follow the guided instructions step by step.
+
+. Think carefully about how to name your app. It *cannot* be changed later on. We recommend naming it *ownCloud for Teams* for easy identification.
+
+. The ID *must* be the APPLICATION-ID from Microsoft Azure and the URL *must* be your owncloud installation's URL.
+// from where can we get this data
+
+. After performing all the steps, click the download button and store the generated zip file locally.
+
+. Go back to the app section of Microsoft Teams and upload the generated zip file to your organizations app catalogue. Follow the {publish-custom-app-url}[Publish a custom app by uploading an app package] guide for more information.
+
+. The new app is now available to users in your organization's app catalog.
+
+. See the {manage-apps-url}[Manage your apps in the Microsoft Teams admin center] and {teams-app-setup-policies-url}[Manage app setup policies in Microsoft Teams] documents to pin the app and set the order how apps appear or install apps on behalf of users.
+
+// Users should refer to [ref to section in User Guide] for information on how to use ownCloud with Teams.
+
+=== Alternative ownCloud Website Tab
+
+As an alternative creating an Microsoft Teams app, it's also possible to embed ownCloud as a Microsoft Teams tab website. Tabs are Teams-aware webpages embedded in Microsoft Teams. See the {what-are-tabs-url}[What are Microsoft Teams tabs] documentation to read more.
+
+. Press the btn:[+] plus button which you will find in top of the Teams window
++
+image:configuration/integration/ms-teams/add-a-tab-ms-teams.png[,width=80%]
+
+. Search for *website* and add by clicking on it
++
+image:configuration/integration/ms-teams/add-website-ms-teams.png[,width=80%]
+
+. Add a meaningful name and the following URl replacing "your-ownCloud-url" with the URL how you access your owncloud instance
++
+image:configuration/integration/ms-teams/name-and-url-ms-teams.png[,width=80%]
++
+[source,url]
+----
+https://your-ownCloud-url/apps/msteamsbridge
+----
+
+== Support
+
+If you encounter problems with the integration of ownCloud and Teams, please contact us at support@owncloud.com or look for answers to the problems at: central.owncloud.com.
```

Comment:
central runs under .org not .com:
right url ist: https://central.owncloud.org

Reply:
fixed